DBZ-8440 Document Debezium Operator (#6617)
debezium/debezium: documentation/modules/ROOT/pages/operations/debezium-operator.adoc

| No default value
| Redis backed schema history store configuration

| [[debezium-operator-schema-reference-schemahistory-kafka]]<<debezium-operator-schema-reference-schemahistory-kafka, `+kafka+`>>
| <<debezium-operator-schema-reference-kafkaschemahistorystore, `+KafkaSchemaHistoryStore+`>>
| No default value
| Kafka backed schema history store configuration

| [[debezium-operator-schema-reference-schemahistory-jdbc]]<<debezium-operator-schema-reference-schemahistory-jdbc, `+jdbc+`>>
| <<debezium-operator-schema-reference-jdbcschemahistorystore, `+JdbcSchemaHistoryStore+`>>
| No default value
| JDBC backed schema history store configuration

| [[debezium-operator-schema-reference-schemahistory-store]]<<debezium-operator-schema-reference-schemahistory-store, `+store+`>>
| <<debezium-operator-schema-reference-customstore, `+CustomStore+`>>
| No default value
| Arbitrary schema history store configuration

| [[debezium-operator-schema-reference-schemahistory-config]]<<debezium-operator-schema-reference-schemahistory-config, `+config+`>>
| Map
| No default value
| Additional common schema history store configuration properties.
|===

> == Example Configuration

Reviewer: ```suggestion
== Complete Debezium Server configuration example
```

> The following example shows a complete `DebeziumServerSpec` configuration:

Reviewer: ```suggestion
The following example shows a complete {prodname} Server custom resource that incorporates properties from the preceding tables. 
```

> [source,yaml]
> ----

Reviewer: ```suggestion
.Complete {prodname} server custom resource
====
[source,yaml]
```

      database.user: debezium
      database.password: secret
      database.server.id: 184054
      topic.prefix: mysql
      database.include.list: inventory
    offset:
      type: kafka
      config:
        bootstrap.servers: kafka:9092
        topic: debezium-offsets
    schemaHistory:
      type: kafka
      config:
        bootstrap.servers: kafka:9092
        topic: debezium-schema-history

  sink:
    type: kafka
    config:
      bootstrap.servers: kafka:9092

  format:
    key:
      type: json
    value:
      type: json
      config:
        schemas.enable: false

  runtime:
    jmx:
      enabled: true
      port: 1099
    storage:
      data:
        type: persistent
        claimName: debezium-data-pvc
    metrics:
      jmxExporter:
        enabled: true

  transforms:
    - type: io.debezium.transforms.UnwrapFromEnvelope
      config:
        drop.tombstones: false
      predicate: inventory-filter
      negate: false

  predicates:
    inventory-filter:
      type: org.apache.kafka.connect.transforms.predicates.TopicNameMatches
      config:
        pattern: "mysql.inventory.*"

  quarkus:
    config:
      quarkus.log.level: INFO
      quarkus.http.port: 8080

> ----

Reviewer: ```suggestion
====
```

          labels:  # Map<String, String>
            key: value
        imagePullSecrets:  # List
          - name: String
        affinity: Affinity
        securityContext: PodSecurityContext
  quarkus:
    config:
      # quarkus properties
      format:
        value:
          type: String
          config:
            # format properties
      key:
        type: String
        config:
          # format properties
      header:
        type: String
        config:
          # format properties
  transforms:
    - type: String
      predicate: String
      negate: Boolean
      config:
        # transformation properties
  predicates:
    name:
      type: String
      config:
        # predicate properties
  sink:
    type: String
    config:
      # sink properties
  source:
    class: String
    config:
      # source connector properties

=== Configuration Sections

==== Top-Level Properties

[#debezium-operator-schema-reference-debeziumserverspec]
Used in: <<debezium-operator-schema-reference-debeziumserver, `+DebeziumServer+`>>

.DebeziumServerSpec properties
[cols="20%a,25%s,15%a,40%a",options="header"]
|===
| Property | Type | Default | Description

> | [[debezium-operator-schema-reference-debeziumserverspec-image]]<<debezium-operator-schema-reference-debeziumserverspec-image, `+image+`>> | String |  | Image used for Debezium Server container. This property takes precedence over version.

Reviewer: ```suggestion
| [[debezium-operator-schema-reference-debeziumserverspec-image]]<<debezium-operator-schema-reference-debeziumserverspec-image, `+image+`>> 
| String 
|No default value 
| Image used for Debezium Server container. This property takes precedence over the version property.
```
Although I verified that the tables as currently defined in asciidoc do render correctly, to facilitate possible future editing, I'd recommend that you place the markup for each table cell on a separate line, as in the preceding suggestion. 
Also, where a property has no default value, add text that makes that explicit.
